src/dune_rules/context.mli

```diff
@@ -41,59 +41,59 @@ module Env_nodes : sig
     }
 end
 
-type t = private
-  { name : Context_name.t
-  ; kind : Kind.t
-  ; profile : Profile.t (** [true] if this context is used for the .merlin files *)
-  ; merlin : bool
-      (** [Some path/to/foo.exe] if this contexts is for feedback-directed
-          optimization of target path/to/foo.exe *)
-  ; fdo_target_exe : Path.t option
-      (** By default Dune builds and installs dynamically linked foreign
-          archives (usually named [dll*.so]). It is possible to disable this
-          by adding (disable_dynamically_linked_foreign_archives true) to the
-          workspace file, in which case bytecode executables will be built
-          with all foreign archives statically linked into the runtime system. *)
-  ; dynamically_linked_foreign_archives : bool
-      (** If this context is a cross-compilation context, you need another
-          context for building tools used for the compilation that run on the
-          host. *)
-  ; for_host : t option
-      (** [false] if a user explicitly listed this context in the workspace.
-          Controls whether we add artifacts from this context \@install *)
-  ; implicit : bool
-      (** Directory where artifact are stored, for instance "_build/default" *)
-  ; build_dir : Path.Build.t (** env node that this context was initialized with *)
-  ; env_nodes : Env_nodes.t
-  ; path : Path.t list (** [PATH] *)
-  ; ocaml : Ocaml_toolchain.t
-  ; installed_env : Env.t
-  ; findlib_paths : Path.t list
-  ; findlib_toolchain : Context_name.t option (** Misc *)
-  ; default_ocamlpath : Path.t list
-  ; build_context : Build_context.t
-  ; instrument_with : Lib_name.t list
-  }
+type t
```

Comment:
A little more comments would be nice, but doesn't have to be in this PR.

Reply:
Added some.